governance/votes/Trial Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,191 @@
+
+Given the successful launch [poll](https://vote.makerdao.com/polling/QmQmxEZp) the following actions will take place if this proposal passes: 
+
+* Deprecate MCD_FLAP and remove MCD_FLAP from chainlog
+* Deploy FlapperUniV2
+* Set ``hop`` to 1,577 seconds - hop defines the minimum interval between kicks, limiting the frequency of market actions.
+* Set ``want`` to 0.98 - A want of 0.98 means that the purchase price of MKR can be up to 2% worse than the oracle price.
+* Set ``receiver`` to [MCD_PAUSE_PROXY](https://etherscan.io/address/0xbe8e3e3618f7474f8cb1d074a26affef007e98fb) - destination address of the LP tokens purchased by the Smart Burn Engine.
+* Add FlapperJob and add to chainlog
+* Set ``pip`` to TBD
+* [Surplus Lot Size](https://manual.makerdao.com/parameter-index/surplus-auction/param-surplus-lot-size): Reduce ``vow.bump`` by 25,000 DAI from 30,000 DAI to 5,000 DAI
+
+Furthermore, the following stability scope change will be made in accordance with [MIP104](https://mips.makerdao.com/mips/details/MIP104#9-1-1-1-the-surplus-buffer-upper-limit-is-50-million-dai-):
+* [Maximum System Surplus](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer): Reduce ``vow.hump`` by 200 million DAI from 250 million DAI to 50 million DAI
+
+### Adding Cron Jobs to Chainlog
+
+The new Flapper addition [requires updates](https://forum.makerdao.com/t/dsscron-housekeeping-additions/21292) to the [cron jobs](https://github.com/makerdao/dss-cron) (scheduled tasks). This will add the following addresses to the [chainlog](https://chainlog.makerdao.com/) contract. 
+
+**Sequencer**: [0x238b4E35dAed6100C6162fAE4510261f88996EC9](https://etherscan.io/address/0x238b4e35daed6100c6162fae4510261f88996ec9)
+
+| Job | Deployment | Details |
+|--|------|--|
+| **[AutoLineJob](https://github.com/makerdao/dss-cron/blob/master/src/AutoLineJob.sol)**| [0x67AD4000e73579B9725eE3A149F85C4Af0A61361](https://etherscan.io/address/0x67ad4000e73579b9725ee3a149f85c4af0a61361)|thi=1000 bps, tlo=5000 bps| 
+|**[LerpJob](https://github.com/makerdao/dss-cron/blob/master/src/LerpJob.sol)**|[0x8F8f2FC1F0380B9Ff4fE5c3142d0811aC89E32fB](https://etherscan.io/address/0x8f8f2fc1f0380b9ff4fe5c3142d0811ac89e32fb)|maxDuration=1 day|
+|**[D3MJob](https://github.com/makerdao/dss-cron/blob/master/src/D3MJob.sol):**|[0x1Bb799509b0B039345f910dfFb71eEfAc7022323](https://etherscan.io/address/0x1bb799509b0b039345f910dffb71eefac7022323#code)|threshold=500 bps, ttl=10 minutes|
+|**[ClipperMomJob](https://github.com/makerdao/dss-cron/blob/master/src/ClipperMomJob.sol):**|[0xc3A76B34CFBdA7A3a5215629a0B937CBDEC7C71a](https://etherscan.io/address/0xc3a76b34cfbda7a3a5215629a0b937cbdec7c71a)|-|
+|**[OracleJob](https://github.com/makerdao/dss-cron/blob/master/src/OracleJob.sol):**|[0xe717Ec34b2707fc8c226b34be5eae8482d06ED03](https://etherscan.io/address/0xe717ec34b2707fc8c226b34be5eae8482d06ed03)|-|
+
+### Aligned Delegate One-Off Compensation
```

Comment:
Ah, needs adding to summaries and stuff. Make sure to read through the whole thing again if you make changes late in the process.

Reply:
Wait.... This isn't the one-off comp. This is the June compensation.